test/variable_rate_reactions.jl

```diff
@@ -1,48 +1,35 @@
 using DiffEqBiological, DiffEqJump, DiffEqBase, OrdinaryDiffEq, StochasticDiffEq
 using Base.Test
+using tmpMod
```

Comment:
What's this?

Reply:
It has to do with me not really being very good with git and understanding the loading of packages in different orders. Basically I could not use the new reaction_network by loading DiffEqBiological, so I renamed it as a module tmpMod so I can load it and try it. 

This is only a temporary I had there while running the code checking that it worked, forgot to remove it.